common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -205,16 +211,6 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
     }
 
     @Override
-    public Iterable<Entry<V>> entries() {
```

Comment:
Sry I'm a bit late to the discussion on this one.  Do we have any benchmarking to determine if getting rid of this is "acceptable"?  I agree it makes things simpler/cleaner, but wondering what getting rid of the "native" entry will cost us.